### DISCUSSION TITLE
audiobooks: bottom scrub bar / reading progression / interactive timeline - needs improvements

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#1770] [Ideas]
Right now, each audio track (or spine item in the reading order of EPUB text books) occupies a fixed width unit. In other words, reading fragments are spread uniformly, which is a more obvious problem with audiobooks than with text books. Each horizontal interval on the timeline contains its own 0-100% progression. Let's try to improve this UX, starting with AudioBooks, then broaden the scope to text books (i.e. the notion of "positions list" is currently missing in R2 navigator, but we can immediately fix the problem with AudioBooks thanks to the `IAudioPlaybackInfo` composed Locator object returned by the navigator)

Follow-up to PR https://github.com/readium/readium-desktop/pull/938

**Comment by llemeurfr** [comment]
Pls check https://github.com/readium/architecture/blob/master/models/locators/other/timeline.md, I wrote it long time ago, may still be useful.

**Comment by danielweck** [comment]
I think we can keep the bottom scrub bar for audiobooks, to offer quick random access to reading order items (individual audio files), as an alternative to previous/next sequential access.

however, the previous/next buttons are redundant with the audio controls displayed in the navigator Ui.

also note that full screen removes the bottom bar (there is already a bug filed in the tracker about the lack of Ui in this mode, which supposes users have access to keyboard ... not the case in touch kiosk mode)
